jenkins_job_manager/cli.py

```diff
@@ -121,7 +121,9 @@ def check_auth(obj: JenkinsJobManager):
         raise click.exceptions.Exit(1)
 
 
-def handle_plan_report(obj: JenkinsJobManager, use_pager=True) -> bool:
+def handle_plan_report(
+    obj: JenkinsJobManager, use_pager=True, report_format="default"
+) -> bool:
```

Comment:
Since this function largely exists to decorate output for console output, we probably want to bypass it entirely for "machine readable" formats like json or yaml